### ISSUE TITLE
Extension rejected from Marketplace during automated test

### ISSUE THREAD

**Issue by dimdimich** [swissup/module-easytabs#12]
```
Environment: PHP 7.0.31 Magento 2.2 CE
php bin/magento setup:upgrade 
Cache cleared successfully
File system cleanup:
/var/www/html/generated/code/Composer
/var/www/html/generated/code/Magento
/var/www/html/generated/code/Symfony
The directory '/var/www/html/generated/metadata/' doesn't exist - skipping cleanup
Updating modules:
Schema creation/updates:
Module 'Magento_Store':
Module 'Magento_Directory':
Module 'Magento_Eav':
Module 'Magento_Theme':
Module 'Magento_Cms':
Module 'Magento_Customer':
Module 'Magento_AdminNotification':
Module 'Magento_Indexer':
Module 'Magento_AdvancedPricingImportExport':
Module 'Magento_Config':
Module 'Magento_Backend':
Module 'Magento_Catalog':
Module 'Magento_Payment':
Module 'Magento_Backup':
Module 'Magento_Quote':
Module 'Magento_Bundle':
Module 'Magento_BundleImportExport':
Module 'Magento_CacheInvalidate':
Module 'Magento_Rule':
Module 'Magento_SalesSequence':
Module 'Magento_Security':
Module 'Magento_CatalogImportExport':
Module 'Magento_Sales':
Module 'Magento_Cron':
Module 'Magento_Msrp':
Module 'Magento_Search':
Module 'Magento_CatalogUrlRewrite':
Module 'Magento_Widget':
Module 'Magento_CatalogInventory':
Module 'Magento_Checkout':
Module 'Magento_Vault':
Module 'Magento_CmsUrlRewrite':
Module 'Magento_User':
Module 'Magento_ConfigurableImportExport':
Module 'Magento_ConfigurableProduct':
Module 'Magento_ConfigurableProductSales':
Module 'Magento_Contact':
Module 'Magento_Cookie':
Module 'Magento_Email':
Module 'Magento_CurrencySymbol':
Module 'Magento_Paypal':
Module 'Magento_Integration':
Module 'Magento_CustomerImportExport':
Module 'Magento_Deploy':
Module 'Magento_Developer':
Module 'Magento_Dhl':
Module 'Amazon_Core':
Module 'Magento_Downloadable':
Module 'Magento_ImportExport':
Module 'Magento_Authorizenet':
Module 'Magento_Captcha':
Module 'Magento_EncryptionKey':
Module 'Magento_Fedex':
Module 'Magento_GiftMessage':
Module 'Magento_GoogleAdwords':
Module 'Magento_GoogleAnalytics':
Module 'Magento_Ui':
Module 'Magento_GroupedImportExport':
Module 'Magento_GroupedProduct':
Module 'Magento_DownloadableImportExport':
Module 'Magento_Newsletter':
Module 'Magento_InstantPurchase':
Module 'Magento_Analytics':
Module 'Magento_LayeredNavigation':
Module 'Magento_Marketplace':
Module 'Magento_MediaStorage':
Module 'Magento_CatalogRule':
Module 'Magento_Multishipping':
Module 'Magento_NewRelicReporting':
Module 'Magento_ProductAlert':
Module 'Magento_OfflinePayments':
Module 'Magento_SalesRule':
Module 'Magento_PageCache':
Module 'Magento_Braintree':
Module 'Klarna_Core':
Module 'Magento_Persistent':
Module 'Magento_Reports':
Module 'Magento_ProductVideo':
Module 'Amazon_Login':
Module 'Magento_QuoteAnalytics':
Module 'Magento_ReleaseNotification':
Module 'Magento_Review':
Module 'Magento_RequireJs':
Module 'Magento_SendFriend':
Module 'Magento_ReviewAnalytics':
Module 'Magento_Robots':
Module 'Magento_Rss':
Module 'Magento_CatalogRuleConfigurable':
Module 'Amazon_Payment':
Module 'Magento_SalesAnalytics':
Module 'Magento_SalesInventory':
Module 'Magento_OfflineShipping':
Module 'Klarna_Ordermanagement':
Module 'Magento_UrlRewrite':
Module 'Magento_CatalogSearch':
Module 'Magento_CustomerAnalytics':
Module 'Magento_Shipping':
Module 'Magento_Wishlist':
Module 'Magento_Signifyd':
Module 'Magento_Sitemap':
Module 'Magento_CheckoutAgreements':
Module 'Magento_Webapi':
Module 'Magento_SwaggerWebapi':
Module 'Magento_Swatches':
Module 'Magento_SwatchesLayeredNavigation':
Module 'Magento_Tax':
Module 'Magento_TaxImportExport':
Module 'Magento_Authorization':
Module 'Magento_Translation':
Module 'Magento_GoogleOptimizer':
Module 'Magento_Ups':
Module 'Magento_SampleData':
Module 'Magento_CatalogAnalytics':
Module 'Magento_Usps':
Module 'Magento_Variable':
Module 'Klarna_Kp':
Module 'Magento_Version':
Module 'Magento_Swagger':
Module 'Magento_WebapiSecurity':
Module 'Magento_Weee':
Module 'Magento_CatalogWidget':
Module 'Dotdigitalgroup_Email':
Module 'Magento_WishlistAnalytics':
Module 'Swissup_Core':
Upgrading schema... 
Module 'Swissup_Easytabs':
Installing schema... Upgrading schema... 
Module 'Temando_Shipping':
Module 'Vertex_Tax':
Schema post-updates:
Module 'Magento_Store':
Module 'Magento_Directory':
Module 'Magento_Eav':
Module 'Magento_Theme':
Module 'Magento_Cms':
Module 'Magento_Customer':
Module 'Magento_AdminNotification':
Module 'Magento_Indexer':
Running schema recurring...
Module 'Magento_AdvancedPricingImportExport':
Module 'Magento_Config':
Module 'Magento_Backend':
Module 'Magento_Catalog':
Running schema recurring...
Module 'Magento_Payment':
Module 'Magento_Backup':
Module 'Magento_Quote':
Module 'Magento_Bundle':
Running schema recurring...
Module 'Magento_BundleImportExport':
Module 'Magento_CacheInvalidate':
Module 'Magento_Rule':
Module 'Magento_SalesSequence':
Module 'Magento_Security':
Module 'Magento_CatalogImportExport':
Module 'Magento_Sales':
Module 'Magento_Cron':
Running schema recurring...
Module 'Magento_Msrp':
Module 'Magento_Search':
Module 'Magento_CatalogUrlRewrite':
Running schema recurring...
Module 'Magento_Widget':
Module 'Magento_CatalogInventory':
Running schema recurring...
Module 'Magento_Checkout':
Module 'Magento_Vault':
Module 'Magento_CmsUrlRewrite':
Module 'Magento_User':
Module 'Magento_ConfigurableImportExport':
Module 'Magento_ConfigurableProduct':
Running schema recurring...
Module 'Magento_ConfigurableProductSales':
Module 'Magento_Contact':
Module 'Magento_Cookie':
Module 'Magento_Email':
Module 'Magento_CurrencySymbol':
Module 'Magento_Paypal':
Module 'Magento_Integration':
Running schema recurring...
Module 'Magento_CustomerImportExport':
Module 'Magento_Deploy':
Module 'Magento_Developer':
Module 'Magento_Dhl':
Module 'Amazon_Core':
Module 'Magento_Downloadable':
Module 'Magento_ImportExport':
Module 'Magento_Authorizenet':
Module 'Magento_Captcha':
Module 'Magento_EncryptionKey':
Module 'Magento_Fedex':
Module 'Magento_GiftMessage':
Module 'Magento_GoogleAdwords':
Module 'Magento_GoogleAnalytics':
Module 'Magento_Ui':
Module 'Magento_GroupedImportExport':
Module 'Magento_GroupedProduct':
Module 'Magento_DownloadableImportExport':
Module 'Magento_Newsletter':
Module 'Magento_InstantPurchase':
Module 'Magento_Analytics':
Module 'Magento_LayeredNavigation':
Module 'Magento_Marketplace':
Module 'Magento_MediaStorage':
Module 'Magento_CatalogRule':
Module 'Magento_Multishipping':
Module 'Magento_NewRelicReporting':
Module 'Magento_ProductAlert':
Running schema recurring...
Module 'Magento_OfflinePayments':
Module 'Magento_SalesRule':
Module 'Magento_PageCache':
Module 'Magento_Braintree':
Module 'Klarna_Core':
Module 'Magento_Persistent':
Module 'Magento_Reports':
Running schema recurring...
Module 'Magento_ProductVideo':
Module 'Amazon_Login':
Module 'Magento_QuoteAnalytics':
Module 'Magento_ReleaseNotification':
Module 'Magento_Review':
Module 'Magento_RequireJs':
Module 'Magento_SendFriend':
Module 'Magento_ReviewAnalytics':
Module 'Magento_Robots':
Module 'Magento_Rss':
Module 'Magento_CatalogRuleConfigurable':
Module 'Amazon_Payment':
Module 'Magento_SalesAnalytics':
Module 'Magento_SalesInventory':
Module 'Magento_OfflineShipping':
Module 'Klarna_Ordermanagement':
Module 'Magento_UrlRewrite':
Module 'Magento_CatalogSearch':
Module 'Magento_CustomerAnalytics':
Module 'Magento_Shipping':
Module 'Magento_Wishlist':
Running schema recurring...
Module 'Magento_Signifyd':
Module 'Magento_Sitemap':
Module 'Magento_CheckoutAgreements':
Module 'Magento_Webapi':
Module 'Magento_SwaggerWebapi':
Module 'Magento_Swatches':
Module 'Magento_SwatchesLayeredNavigation':
Module 'Magento_Tax':
Module 'Magento_TaxImportExport':
Module 'Magento_Authorization':
Module 'Magento_Translation':
Module 'Magento_GoogleOptimizer':
Module 'Magento_Ups':
Module 'Magento_SampleData':
Module 'Magento_CatalogAnalytics':
Module 'Magento_Usps':
Module 'Magento_Variable':
Module 'Klarna_Kp':
Module 'Magento_Version':
Module 'Magento_Swagger':
Module 'Magento_WebapiSecurity':
Module 'Magento_Weee':
Running schema recurring...
Module 'Magento_CatalogWidget':
Module 'Dotdigitalgroup_Email':
Running schema recurring...
Module 'Magento_WishlistAnalytics':
Module 'Swissup_Core':
Module 'Swissup_Easytabs':
Module 'Temando_Shipping':
Module 'Vertex_Tax':
Data install/update:
Module 'Magento_Store':
Module 'Magento_Directory':
Module 'Magento_Eav':
Module 'Magento_Theme':
Module 'Magento_Cms':
Module 'Magento_Customer':
Module 'Magento_AdminNotification':
Module 'Magento_Indexer':
Module 'Magento_AdvancedPricingImportExport':
Module 'Magento_Config':
Module 'Magento_Backend':
Module 'Magento_Catalog':
Module 'Magento_Payment':
Module 'Magento_Backup':
Module 'Magento_Quote':
Module 'Magento_Bundle':
Module 'Magento_BundleImportExport':
Module 'Magento_CacheInvalidate':
Module 'Magento_Rule':
Module 'Magento_SalesSequence':
Module 'Magento_Security':
Module 'Magento_CatalogImportExport':
Module 'Magento_Sales':
Module 'Magento_Cron':
Module 'Magento_Msrp':
Module 'Magento_Search':
Module 'Magento_CatalogUrlRewrite':
Module 'Magento_Widget':
Module 'Magento_CatalogInventory':
Module 'Magento_Checkout':
Module 'Magento_Vault':
Module 'Magento_CmsUrlRewrite':
Module 'Magento_User':
Module 'Magento_ConfigurableImportExport':
Module 'Magento_ConfigurableProduct':
Module 'Magento_ConfigurableProductSales':
Module 'Magento_Contact':
Module 'Magento_Cookie':
Module 'Magento_Email':
Module 'Magento_CurrencySymbol':
Module 'Magento_Paypal':
Module 'Magento_Integration':
Module 'Magento_CustomerImportExport':
Module 'Magento_Deploy':
Module 'Magento_Developer':
Module 'Magento_Dhl':
Module 'Amazon_Core':
Module 'Magento_Downloadable':
Module 'Magento_ImportExport':
Module 'Magento_Authorizenet':
Module 'Magento_Captcha':
Module 'Magento_EncryptionKey':
Module 'Magento_Fedex':
Module 'Magento_GiftMessage':
Module 'Magento_GoogleAdwords':
Module 'Magento_GoogleAnalytics':
Module 'Magento_Ui':
Module 'Magento_GroupedImportExport':
Module 'Magento_GroupedProduct':
Module 'Magento_DownloadableImportExport':
Module 'Magento_Newsletter':
Module 'Magento_InstantPurchase':
Module 'Magento_Analytics':
Module 'Magento_LayeredNavigation':
Module 'Magento_Marketplace':
Module 'Magento_MediaStorage':
Module 'Magento_CatalogRule':
Module 'Magento_Multishipping':
Module 'Magento_NewRelicReporting':
Module 'Magento_ProductAlert':
Module 'Magento_OfflinePayments':
Module 'Magento_SalesRule':
Module 'Magento_PageCache':
Module 'Magento_Braintree':
Module 'Klarna_Core':
Module 'Magento_Persistent':
Module 'Magento_Reports':
Module 'Magento_ProductVideo':
Module 'Amazon_Login':
Module 'Magento_QuoteAnalytics':
Module 'Magento_ReleaseNotification':
Module 'Magento_Review':
Module 'Magento_RequireJs':
Module 'Magento_SendFriend':
Module 'Magento_ReviewAnalytics':
Module 'Magento_Robots':
Module 'Magento_Rss':
Module 'Magento_CatalogRuleConfigurable':
Module 'Amazon_Payment':
Module 'Magento_SalesAnalytics':
Module 'Magento_SalesInventory':
Module 'Magento_OfflineShipping':
Module 'Klarna_Ordermanagement':
Module 'Magento_UrlRewrite':
Module 'Magento_CatalogSearch':
Module 'Magento_CustomerAnalytics':
Module 'Magento_Shipping':
Module 'Magento_Wishlist':
Module 'Magento_Signifyd':
Module 'Magento_Sitemap':
Module 'Magento_CheckoutAgreements':
Module 'Magento_Webapi':
Module 'Magento_SwaggerWebapi':
Module 'Magento_Swatches':
Module 'Magento_SwatchesLayeredNavigation':
Module 'Magento_Tax':
Module 'Magento_TaxImportExport':
Module 'Magento_Authorization':
Module 'Magento_Translation':
Module 'Magento_GoogleOptimizer':
Module 'Magento_Ups':
Module 'Magento_SampleData':
Module 'Magento_CatalogAnalytics':
Module 'Magento_Usps':
Module 'Magento_Variable':
Module 'Klarna_Kp':
Module 'Magento_Version':
Module 'Magento_Swagger':
Module 'Magento_WebapiSecurity':
Module 'Magento_Weee':
Module 'Magento_CatalogWidget':
Module 'Dotdigitalgroup_Email':
Module 'Magento_WishlistAnalytics':
Module 'Swissup_Core':
Module 'Swissup_Easytabs':
Installing data... Area code not set: Area code must be set before starting a session.
Environment: PHP 7.1.22 Magento 2.2 CE
php bin/magento setup:upgrade 
Cache cleared successfully
File system cleanup:
/var/www/html/generated/code/Composer
/var/www/html/generated/code/Magento
/var/www/html/generated/code/Symfony
The directory '/var/www/html/generated/metadata/' doesn't exist - skipping cleanup
Updating modules:
Schema creation/updates:
Module 'Magento_Store':
Module 'Magento_Directory':
Module 'Magento_Eav':
Module 'Magento_Theme':
Module 'Magento_Cms':
Module 'Magento_Customer':
Module 'Magento_AdminNotification':
Module 'Magento_Indexer':
Module 'Magento_AdvancedPricingImportExport':
Module 'Magento_Config':
Module 'Magento_Backend':
Module 'Magento_Catalog':
Module 'Magento_Payment':
Module 'Magento_Backup':
Module 'Magento_Quote':
Module 'Magento_Bundle':
Module 'Magento_BundleImportExport':
Module 'Magento_CacheInvalidate':
Module 'Magento_Rule':
Module 'Magento_SalesSequence':
Module 'Magento_Security':
Module 'Magento_CatalogImportExport':
Module 'Magento_Sales':
Module 'Magento_Cron':
Module 'Magento_Msrp':
Module 'Magento_Search':
Module 'Magento_CatalogUrlRewrite':
Module 'Magento_Widget':
Module 'Magento_CatalogInventory':
Module 'Magento_Checkout':
Module 'Magento_Vault':
Module 'Magento_CmsUrlRewrite':
Module 'Magento_User':
Module 'Magento_ConfigurableImportExport':
Module 'Magento_ConfigurableProduct':
Module 'Magento_ConfigurableProductSales':
Module 'Magento_Contact':
Module 'Magento_Cookie':
Module 'Magento_Email':
Module 'Magento_CurrencySymbol':
Module 'Magento_Paypal':
Module 'Magento_Integration':
Module 'Magento_CustomerImportExport':
Module 'Magento_Deploy':
Module 'Magento_Developer':
Module 'Magento_Dhl':
Module 'Amazon_Core':
Module 'Magento_Downloadable':
Module 'Magento_ImportExport':
Module 'Magento_Authorizenet':
Module 'Magento_Captcha':
Module 'Magento_EncryptionKey':
Module 'Magento_Fedex':
Module 'Magento_GiftMessage':
Module 'Magento_GoogleAdwords':
Module 'Magento_GoogleAnalytics':
Module 'Magento_Ui':
Module 'Magento_GroupedImportExport':
Module 'Magento_GroupedProduct':
Module 'Magento_DownloadableImportExport':
Module 'Magento_Newsletter':
Module 'Magento_InstantPurchase':
Module 'Magento_Analytics':
Module 'Magento_LayeredNavigation':
Module 'Magento_Marketplace':
Module 'Magento_MediaStorage':
Module 'Magento_CatalogRule':
Module 'Magento_Multishipping':
Module 'Magento_NewRelicReporting':
Module 'Magento_ProductAlert':
Module 'Magento_OfflinePayments':
Module 'Magento_SalesRule':
Module 'Magento_PageCache':
Module 'Magento_Braintree':
Module 'Klarna_Core':
Module 'Magento_Persistent':
Module 'Magento_Reports':
Module 'Magento_ProductVideo':
Module 'Amazon_Login':
Module 'Magento_QuoteAnalytics':
Module 'Magento_ReleaseNotification':
Module 'Magento_Review':
Module 'Magento_RequireJs':
Module 'Magento_SendFriend':
Module 'Magento_ReviewAnalytics':
Module 'Magento_Robots':
Module 'Magento_Rss':
Module 'Magento_CatalogRuleConfigurable':
Module 'Amazon_Payment':
Module 'Magento_SalesAnalytics':
Module 'Magento_SalesInventory':
Module 'Magento_OfflineShipping':
Module 'Klarna_Ordermanagement':
Module 'Magento_UrlRewrite':
Module 'Magento_CatalogSearch':
Module 'Magento_CustomerAnalytics':
Module 'Magento_Shipping':
Module 'Magento_Wishlist':
Module 'Magento_Signifyd':
Module 'Magento_Sitemap':
Module 'Magento_CheckoutAgreements':
Module 'Magento_Webapi':
Module 'Magento_SwaggerWebapi':
Module 'Magento_Swatches':
Module 'Magento_SwatchesLayeredNavigation':
Module 'Magento_Tax':
Module 'Magento_TaxImportExport':
Module 'Magento_Authorization':
Module 'Magento_Translation':
Module 'Magento_GoogleOptimizer':
Module 'Magento_Ups':
Module 'Magento_SampleData':
Module 'Magento_CatalogAnalytics':
Module 'Magento_Usps':
Module 'Magento_Variable':
Module 'Klarna_Kp':
Module 'Magento_Version':
Module 'Magento_Swagger':
Module 'Magento_WebapiSecurity':
Module 'Magento_Weee':
Module 'Magento_CatalogWidget':
Module 'Dotdigitalgroup_Email':
Module 'Magento_WishlistAnalytics':
Module 'Swissup_Core':
Upgrading schema... 
Module 'Swissup_Easytabs':
Installing schema... Upgrading schema... 
Module 'Temando_Shipping':
Module 'Vertex_Tax':
Schema post-updates:
Module 'Magento_Store':
Module 'Magento_Directory':
Module 'Magento_Eav':
Module 'Magento_Theme':
Module 'Magento_Cms':
Module 'Magento_Customer':
Module 'Magento_AdminNotification':
Module 'Magento_Indexer':
Running schema recurring...
Module 'Magento_AdvancedPricingImportExport':
Module 'Magento_Config':
Module 'Magento_Backend':
Module 'Magento_Catalog':
Running schema recurring...
Module 'Magento_Payment':
Module 'Magento_Backup':
Module 'Magento_Quote':
Module 'Magento_Bundle':
Running schema recurring...
Module 'Magento_BundleImportExport':
Module 'Magento_CacheInvalidate':
Module 'Magento_Rule':
Module 'Magento_SalesSequence':
Module 'Magento_Security':
Module 'Magento_CatalogImportExport':
Module 'Magento_Sales':
Module 'Magento_Cron':
Running schema recurring...
Module 'Magento_Msrp':
Module 'Magento_Search':
Module 'Magento_CatalogUrlRewrite':
Running schema recurring...
Module 'Magento_Widget':
Module 'Magento_CatalogInventory':
Running schema recurring...
Module 'Magento_Checkout':
Module 'Magento_Vault':
Module 'Magento_CmsUrlRewrite':
Module 'Magento_User':
Module 'Magento_ConfigurableImportExport':
Module 'Magento_ConfigurableProduct':
Running schema recurring...
Module 'Magento_ConfigurableProductSales':
Module 'Magento_Contact':
Module 'Magento_Cookie':
Module 'Magento_Email':
Module 'Magento_CurrencySymbol':
Module 'Magento_Paypal':
Module 'Magento_Integration':
Running schema recurring...
Module 'Magento_CustomerImportExport':
Module 'Magento_Deploy':
Module 'Magento_Developer':
Module 'Magento_Dhl':
Module 'Amazon_Core':
Module 'Magento_Downloadable':
Module 'Magento_ImportExport':
Module 'Magento_Authorizenet':
Module 'Magento_Captcha':
Module 'Magento_EncryptionKey':
Module 'Magento_Fedex':
Module 'Magento_GiftMessage':
Module 'Magento_GoogleAdwords':
Module 'Magento_GoogleAnalytics':
Module 'Magento_Ui':
Module 'Magento_GroupedImportExport':
Module 'Magento_GroupedProduct':
Module 'Magento_DownloadableImportExport':
Module 'Magento_Newsletter':
Module 'Magento_InstantPurchase':
Module 'Magento_Analytics':
Module 'Magento_LayeredNavigation':
Module 'Magento_Marketplace':
Module 'Magento_MediaStorage':
Module 'Magento_CatalogRule':
Module 'Magento_Multishipping':
Module 'Magento_NewRelicReporting':
Module 'Magento_ProductAlert':
Running schema recurring...
Module 'Magento_OfflinePayments':
Module 'Magento_SalesRule':
Module 'Magento_PageCache':
Module 'Magento_Braintree':
Module 'Klarna_Core':
Module 'Magento_Persistent':
Module 'Magento_Reports':
Running schema recurring...
Module 'Magento_ProductVideo':
Module 'Amazon_Login':
Module 'Magento_QuoteAnalytics':
Module 'Magento_ReleaseNotification':
Module 'Magento_Review':
Module 'Magento_RequireJs':
Module 'Magento_SendFriend':
Module 'Magento_ReviewAnalytics':
Module 'Magento_Robots':
Module 'Magento_Rss':
Module 'Magento_CatalogRuleConfigurable':
Module 'Amazon_Payment':
Module 'Magento_SalesAnalytics':
Module 'Magento_SalesInventory':
Module 'Magento_OfflineShipping':
Module 'Klarna_Ordermanagement':
Module 'Magento_UrlRewrite':
Module 'Magento_CatalogSearch':
Module 'Magento_CustomerAnalytics':
Module 'Magento_Shipping':
Module 'Magento_Wishlist':
Running schema recurring...
Module 'Magento_Signifyd':
Module 'Magento_Sitemap':
Module 'Magento_CheckoutAgreements':
Module 'Magento_Webapi':
Module 'Magento_SwaggerWebapi':
Module 'Magento_Swatches':
Module 'Magento_SwatchesLayeredNavigation':
Module 'Magento_Tax':
Module 'Magento_TaxImportExport':
Module 'Magento_Authorization':
Module 'Magento_Translation':
Module 'Magento_GoogleOptimizer':
Module 'Magento_Ups':
Module 'Magento_SampleData':
Module 'Magento_CatalogAnalytics':
Module 'Magento_Usps':
Module 'Magento_Variable':
Module 'Klarna_Kp':
Module 'Magento_Version':
Module 'Magento_Swagger':
Module 'Magento_WebapiSecurity':
Module 'Magento_Weee':
Running schema recurring...
Module 'Magento_CatalogWidget':
Module 'Dotdigitalgroup_Email':
Running schema recurring...
Module 'Magento_WishlistAnalytics':
Module 'Swissup_Core':
Module 'Swissup_Easytabs':
Module 'Temando_Shipping':
Module 'Vertex_Tax':
Data install/update:
Module 'Magento_Store':
Module 'Magento_Directory':
Module 'Magento_Eav':
Module 'Magento_Theme':
Module 'Magento_Cms':
Module 'Magento_Customer':
Module 'Magento_AdminNotification':
Module 'Magento_Indexer':
Module 'Magento_AdvancedPricingImportExport':
Module 'Magento_Config':
Module 'Magento_Backend':
Module 'Magento_Catalog':
Module 'Magento_Payment':
Module 'Magento_Backup':
Module 'Magento_Quote':
Module 'Magento_Bundle':
Module 'Magento_BundleImportExport':
Module 'Magento_CacheInvalidate':
Module 'Magento_Rule':
Module 'Magento_SalesSequence':
Module 'Magento_Security':
Module 'Magento_CatalogImportExport':
Module 'Magento_Sales':
Module 'Magento_Cron':
Module 'Magento_Msrp':
Module 'Magento_Search':
Module 'Magento_CatalogUrlRewrite':
Module 'Magento_Widget':
Module 'Magento_CatalogInventory':
Module 'Magento_Checkout':
Module 'Magento_Vault':
Module 'Magento_CmsUrlRewrite':
Module 'Magento_User':
Module 'Magento_ConfigurableImportExport':
Module 'Magento_ConfigurableProduct':
Module 'Magento_ConfigurableProductSales':
Module 'Magento_Contact':
Module 'Magento_Cookie':
Module 'Magento_Email':
Module 'Magento_CurrencySymbol':
Module 'Magento_Paypal':
Module 'Magento_Integration':
Module 'Magento_CustomerImportExport':
Module 'Magento_Deploy':
Module 'Magento_Developer':
Module 'Magento_Dhl':
Module 'Amazon_Core':
Module 'Magento_Downloadable':
Module 'Magento_ImportExport':
Module 'Magento_Authorizenet':
Module 'Magento_Captcha':
Module 'Magento_EncryptionKey':
Module 'Magento_Fedex':
Module 'Magento_GiftMessage':
Module 'Magento_GoogleAdwords':
Module 'Magento_GoogleAnalytics':
Module 'Magento_Ui':
Module 'Magento_GroupedImportExport':
Module 'Magento_GroupedProduct':
Module 'Magento_DownloadableImportExport':
Module 'Magento_Newsletter':
Module 'Magento_InstantPurchase':
Module 'Magento_Analytics':
Module 'Magento_LayeredNavigation':
Module 'Magento_Marketplace':
Module 'Magento_MediaStorage':
Module 'Magento_CatalogRule':
Module 'Magento_Multishipping':
Module 'Magento_NewRelicReporting':
Module 'Magento_ProductAlert':
Module 'Magento_OfflinePayments':
Module 'Magento_SalesRule':
Module 'Magento_PageCache':
Module 'Magento_Braintree':
Module 'Klarna_Core':
Module 'Magento_Persistent':
Module 'Magento_Reports':
Module 'Magento_ProductVideo':
Module 'Amazon_Login':
Module 'Magento_QuoteAnalytics':
Module 'Magento_ReleaseNotification':
Module 'Magento_Review':
Module 'Magento_RequireJs':
Module 'Magento_SendFriend':
Module 'Magento_ReviewAnalytics':
Module 'Magento_Robots':
Module 'Magento_Rss':
Module 'Magento_CatalogRuleConfigurable':
Module 'Amazon_Payment':
Module 'Magento_SalesAnalytics':
Module 'Magento_SalesInventory':
Module 'Magento_OfflineShipping':
Module 'Klarna_Ordermanagement':
Module 'Magento_UrlRewrite':
Module 'Magento_CatalogSearch':
Module 'Magento_CustomerAnalytics':
Module 'Magento_Shipping':
Module 'Magento_Wishlist':
Module 'Magento_Signifyd':
Module 'Magento_Sitemap':
Module 'Magento_CheckoutAgreements':
Module 'Magento_Webapi':
Module 'Magento_SwaggerWebapi':
Module 'Magento_Swatches':
Module 'Magento_SwatchesLayeredNavigation':
Module 'Magento_Tax':
Module 'Magento_TaxImportExport':
Module 'Magento_Authorization':
Module 'Magento_Translation':
Module 'Magento_GoogleOptimizer':
Module 'Magento_Ups':
Module 'Magento_SampleData':
Module 'Magento_CatalogAnalytics':
Module 'Magento_Usps':
Module 'Magento_Variable':
Module 'Klarna_Kp':
Module 'Magento_Version':
Module 'Magento_Swagger':
Module 'Magento_WebapiSecurity':
Module 'Magento_Weee':
Module 'Magento_CatalogWidget':
Module 'Dotdigitalgroup_Email':
Module 'Magento_WishlistAnalytics':
Module 'Swissup_Core':
Module 'Swissup_Easytabs':
Installing data... Area code not set: Area code must be set before starting a session
```

**Comment by dimdimich** [comment]
You can repack 1.5.0 release with fix for this installation problem.

**Comment by tangar76** [comment]
Version 1.5.0 regenerated with this issue solved.

**Comment by dimdimich** [comment]
Still have that issue at 
Environment: PHP 7.1.22 Magento 2.2 CE

Other configs works fines.

Are you sure that this issue is fixed? Can you generate 1.5.1 release? Probably it's issue because of using same version.
